libraries/SRV_Channel/SRV_Channel.cpp

```diff
@@ -221,3 +221,14 @@ bool SRV_Channel::should_e_stop(SRV_Channel::Aux_servo_function_t function)
             (function >= SRV_Channel::k_boost_throttle && function <= SRV_Channel::k_motor12) ||
             function == k_engine_run_enable);
 }
+
+// return true if function is for a control surface
+bool SRV_Channel::is_control_surface(SRV_Channel::Aux_servo_function_t function)
+{
+    return ((function >= SRV_Channel::k_flap && function <= SRV_Channel::k_aileron) ||
```

Comment:
The >= method used here is not maintainable. Please use a switch.

Reply:
@magicrub Can I use == operator separately for each control surface in a range (instead of >= and <= )? Will it be fine?

Reply:
Please use a switch. There are plenty of examples of that in the servo library.

Reply:
Done! Please check if I did it properly.